tests/FINDOLOGIC/Tests/Objects/JsonResponseTest.php

```diff
@@ -44,16 +45,16 @@ public function testResponseWillReturnExpectedLabels()
     public function testResponseWillReturnExpectedBlocks()
     {
         $expectedBlocks = [
-            'suggest',
-            'suggest',
-            'vendor',
-            'vendor',
-            'cat',
-            'cat',
-            'product',
-            'product',
-            'product',
-            'product',
+            BlockType::SUGGEST_BLOCK,
```

Comment:
Good idea, but please use strings instead of constants. If somebody was about to change the value of the constants, we would not notice it and the tests would still be ok with it.

This applies to all strings that were changed to constants after the change.